docker-compose.yml

```diff
@@ -35,6 +35,13 @@ services:
     networks:
       - generic-network
       - metrics-network
+  hom-bot:
+    container_name: hom-bot
+    image: wiseoldman/hom:latest
+    env_file:
+      - ./.env
+    networks:
```

Comment:
I don't think this bot requires any networks, it doesn't rely on any other service in this compose, right?